package.json

```
@@ -20,6 +20,7 @@
     "gas-report": "pnpm run --recursive --parallel gas-report",
     "lint": "pnpm prettier:check && eslint . --ext .ts --ext .tsx",
     "prepare": "husky install && (forge --version || pnpm foundryup)",
+    "prepare-vercel": "(forge --version || pnpm foundryup) && ln -sf /vercel/.foundry/bin/* node_modules/.bin/ && forge --version",
```

small preference for this to be
```suggestion
    "vercel:prepare": "(forge --version || pnpm foundryup) && ln -sf /vercel/.foundry/bin/* node_modules/.bin/ && forge --version",
```

---

oh true, aligns with the rest of the script names, missed it, updated.

---

ahh I was hoping for `vercel` as the prefix (as that's the "context" in which its run) rather than `prepare` which kinda feels like part of the other prepare steps (like we do for `build` and `clean` and `test`)

---

ah damn, should have just committed the suggestion and missed it, will change in another PR, my bad